Gopkg.toml

```diff
@@ -535,3 +535,7 @@
 [[constraint]]
   name = "k8s.io/apiextensions-apiserver"
   revision = "4d05e43ad98c1edcf2f52291685bd9bbc12fd2cd"
+
+[[constraint]]
```

Comment:
This is a huge dependency to bring in just for the ContainerStarted const.
We can just define our own const and avoid the dependency. Fair enough to bring in the dependency if we use more in the future.

Reply:
removed this dep, and created a new file named events.go to define const there and moved events related methods there from k8s.go as well